### DISCUSSION TITLE
Slow analogRead()

### DISCUSSION THREAD

**Discussion by RudolfAtHome** [dbuezas/lgt8fx#151] [Q&A]
In the benchmarks I found that the LGT8F is typically 2 ... 3 times faster than a 328P. 

But there is one exception: the analogRead (pin) is slower - why?
**ATMega328P: 110 µs**
**LGT8F328P:    167 µs**
I've checked, the number is correct.

**Comment by jayzakk** [comment]
This is already fixed in #31 and will read much faster than Atmega.
The current release does not contain the bugfix.

**Reply by RudolfAtHome** [reply]
Ok - thank you! I can wait for it.
